ui/cypress/integration/Solutions.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Solutions 
+   Scenario: Environment List
+     Given I log in
+     When I go to the solutions list by clicking the solution icon in the sidebar
+     And I go to the Create Envrionments page by clicking the Create a New Environment button
```

Comment:
```suggestion
     And I go to the Create Environments page by clicking the Create a New Environment button
```

Reply:
Sorry for the spelling mistake.
And thank you for the correction!